### PR TITLE
UI/update readme

### PR DESCRIPTION
**Pull request by eao10000**
@xWngl and I have finished cleaning up and updating the READMME.md in the ui directory. We added additional information about the coordinates module while also removing the DFS module from the README.md as it was no longer needed. We were also able to update the how to use section making it easier and more accurate on how to run the ui directory.

### CODE REVIEW

**Review by xWngl — APPROVED** [review]
I'd just add that the old DFS module is now part of the coordinates module

File: src/ui/README.md [hunk 1]
```diff
@@ -11,10 +11,11 @@ are a few currently enabled commands.
...
 
-To compile the library, run `make` inside the ui directory. Example files can be
-found in the ui/examples subdirectory. You can create the binaries for these files
-by running `make examples` to compile the executables for all the examples or
-`make examples/file_name` to compile a specific executable.
+To compile the ui library, run make inside the chiventure/build directory. Example
+files can be found in the src/ui/examples subdirectory from build. You can create
+the binaries for all of these files by running "make" inside the examples 
+subdirectory or using "make file_name" for a specific example. Use "./file_name"
+to run the executable.
```
Comment (MaxineK36) [inline src/ui/README.md:18]: Can we also specify what examples are currently there?

**Review by MaxineK36 — CHANGES_REQUESTED** [review]
Are the `window` and `map` modules still being used? If not, we should note that. Otherwise, looks good modulo my one comment!

**Review by MaxineK36 — APPROVED** [review]
LGTM!

**Comment by eao10000** [comment]
The README.md in the UI module has been updated and implemented. When doing this we noticed that the coordinates section was not completed, so we added the information there, while deleting the unused DFS section. With this the issue is closed.

**Comment by MaxineK36** [comment]
Issue Score: ✔️++

Comments:
Great work! Going forward, please make sure that everyone who contributed to the code is an assignee on the PR, and that they are not also assigned as reviewers.

**Pull request merged**